src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/GlyphRun.cs

```diff
@@ -1269,9 +1269,9 @@ public Rect ComputeInkBoundingBox()
 
             if ((_flags & GlyphRunFlags.CacheInkBounds) != 0)
             {
-                if (_inkBoundingBox != null)
+                if (_inkBoundingBox.HasValue)
                 {
-                    return (Rect)_inkBoundingBox;
+                    return _inkBoundingBox.Value;
```

Comment:
```suggestion
                    return _inkBoundingBox.GetValueOrDefault();
```
`Value` repeats the check for `HasValue` which is already done.

Reply:
@gfoidl Isn't this supposed to be optimized by the JIT in .Net 6 by removing redundant checks when inlining ? Look for "GetValueOrDefault" in the [.Net 6 performance improvements blog post](https://devblogs.microsoft.com/dotnet/performance-improvements-in-net-6/)

Reply:
Oh, you are right, the JIT will do this now for us. Thanks for the reminder! (I'm too used to that pattern with GetValueOrDefault...)

Reply:
@gfoidl It means I need not do any change?

Reply:
@lindexi yeah, leave it as is (it's fine so).
Sorry for the confusion.

Reply:
Thank you @gfoidl